packages/shared/src/types/defaultTheme.ts

```diff
@@ -239,8 +239,8 @@ export interface DocFooter {
 
 export interface SocialLink {
   icon: SocialLinkIcon;
-  mode: 'link' | 'text' | 'img';
-  content: string;
+  mode: 'link' | 'text' | 'img' | 'dom';
+  content: string | HTMLElement;
```

Comment:
It should be more accurate here?
- When `mode` is not `dom`, the type is string. 
- When `mode` is `dom`, the type is string | HTMLElement

Reply:
I'm quite busy today, so messages may come in waves. Sorry for that.